Sources/AnimalKingdomAPI/Generated/Package.resolved

```diff
@@ -1,43 +0,0 @@
-{
```

Comment:
gotta bring this back

Reply:
We don't actually. It looks like it's been .gitignore'd for a while, but was never untracked